genesis/bin/genesis_playback.rs

```diff
@@ -275,8 +282,21 @@ fn main() {
             // Inject account.
             // (storage expiry initialized to 0)
             state.new_contract(&address, balance, nonce, 0);
+            ops += 1;
```

Comment:
This block (increment ops and check if we should commit) could be its own closure that can be called in three different places to avoid duplicating it?

Reply:
agreed, done :+1: